### PR TITLE
Phoenix spark

### PR DESCRIPTION
**Pull request by bpanneton**
Get updated version of phoenix working with updated zeppelin's spark interpreter

### CODE REVIEW

File: magpie/setup/magpie-setup-project-zeppelin [hunk 1]
```diff
@@ -153,9 +153,18 @@ then
     phoenix_jars="${PHOENIX_HOME}/phoenix-${PHOENIX_VERSION}-server.jar,$phoenix_jars"
     phoenix_jars="${PHOENIX_HOME}/phoenix-spark-${PHOENIX_VERSION}.jar,$phoenix_jars"
     phoenix_jars="${PHOENIX_HOME}/phoenix-core-${PHOENIX_VERSION}.jar,$phoenix_jars"
-    phoenix_jars="${PHOENIX_HOME}/phoenix-server-${PHOENIX_VERSION}.jar,$phoenix_jars"
-    phoenix_jars="${PHOENIX_HOME}/phoenix-server-client-${PHOENIX_VERSION}.jar,$phoenix_jars"
-    zeppelin_jars="${zeppelin_jars},${phoenix_jars}"
+    
+    Magpie_vercomp ${PHOENIX_VERSION} 4.8.0
```
Comment (chu11) [inline magpie/setup/magpie-setup-project-zeppelin]: Could you add ```# Returns 0 for ==, 1 for $1 > $2, 2 for $1 < $2``` at the top of the vercomp.  B/c Magpie_vercomp is not intuitive, I try to add it everywhere I use it.

File: magpie/setup/magpie-setup-project-zeppelin [hunk 2]
```diff
@@ -153,9 +153,18 @@ then
...
+    vercomp_result=$?
+    if [ "${vercomp_result}" == "0" ] || [ "${vercomp_result}" == "1" ]
+    then
+        phoenix_jars="${PHOENIX_HOME}/phoenix-queryserver-${PHOENIX_VERSION}.jar,$phoenix_jars"
+        phoenix_jars="${PHOENIX_HOME}/phoenix-queryserver-client-${PHOENIX_VERSION}.jar,$phoenix_jars"
+    else
+        phoenix_jars="${PHOENIX_HOME}/phoenix-server-${PHOENIX_VERSION}.jar,$phoenix_jars"
+        phoenix_jars="${PHOENIX_HOME}/phoenix-server-client-${PHOENIX_VERSION}.jar,$phoenix_jars"
+    fi
+    zeppelin_jars="${zeppelin_jars%,},${phoenix_jars%,}"
```
Comment (chu11) [inline magpie/setup/magpie-setup-project-zeppelin]: I think the code above this is correct and there's no danger of a dangling comma??

**Comment by bpanneton** [comment]
I removed the commas and figured out where the bug was for the zeppelin-spark patch.

**Comment by chu11** [comment]
lgtm.  on a side note, is there any kind of unit test you think could be developed to detect this issue in the future?  I don't know phoenix or zeppelin that well.  The current unit test for zeppelin is just to check if the web server is up.

**Pull request merged**